### ISSUE TITLE
NVDA anounces indent and hanging indent shortcuts but it fails to anounce some other shortcuts that were anounced before.

### ISSUE THREAD

**Issue by jacobchawala** [nvaccess/nvda#12737]
### Steps to reproduce:
According to NVDA 2021.2 beta1, in Microsoft Word, NVDA now announces the result of indent and hanging indent shortcuts when executed. (#6269). Thank you for that. NVDA now announces CTRL+m and CTRL+t as indent and hanging indent shortcuts; but unfortunately, it does not announce other shortcuts that were announced in previous versions of NVDA; for instance:
CTRL+c, CTRL+v, CTRL+x, CTRL+y, and CTRL+z are not announced. 

### Actual behavior:
CTRL+c, CTRL+v, CTRL+x, CTRL+y, and CTRL+z are not announced in NVDA 2021.2 beta1
CTRL+s and FN key were not announced since then

### Expected behavior:
These shortcuts should also be announced as it was in previous versions of NVDA.
CTRL+s should be announced as saved

#### NVDA installed/portable/running from source:
NVDA installer
#### NVDA version:
NVDA 2021.2 beta1
#### Windows version:
Windows 10 professional 64 bits
#### Name and version of other software in use when reproducing the issue:
Microsoft® Word 2019 MSO (16.0.10730.20102) 64-bit 
Product ID: 00414-50000-00000-AA650 Product ID: 00414-50000-00000-AA650
Session ID: 733C0A7B-CF0A-4DBF-A506-57344C3C0056 Session ID: 733C0A7B-CF0A-4DBF-A506-57344C3C0056
#### Other information about your system:
Machine name: DESKTOP-JJC96NM
Machine Id: {7DF14EB0-F3A0-49E1-A882-0E29E93F09A1}
Operating System: Windows 10 Pro 64-bit (10.0, Build 19043) (19041.vb_release.191206-1406)
Language: English (Regional Setting: English)
System Manufacturer: HP
System Model: HP ENVY x360 Convertible 15-cn0xxx
BIOS: InsydeH2O Version 05.12.09F.22 (type: BIOS)
Processor: Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz (8 CPUs), ~2.0GHz
Memory: 16384MB RAM
Available OS Memory: 16218MB RAM

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, mentioned shortcuts were announced except CTRL+s and FN key
#### If add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by seanbudd** [comment]
I cannot reproduce this on `Microsoft® Word for Microsoft 365 MSO (16.0.14228.20216) 64-bit`.
Would you be able to:
1. Share a [log](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps#log-files) of what happens when you perform these shortcuts
2. Update Word and see if the issue still occurs

**Comment by CyrilleB79** [comment]
@jacobchawala you write:
> but unfortunately, it does not announce other shortcuts that were announced in previous versions of NVDA; for instance:
CTRL+c, CTRL+v, CTRL+x, CTRL+y, and CTRL+z are not announced.

According to my experience on Word 2016, these shortcuts were never reported. I can see two possibility for your experience differing from mine:
1. You were previously using an add-on to have it reported and you are not using it anymore in the new NVDA version (maybe due to incompatility with NVDA 2021.1 and above). What is the older version of NVDA where you had these actions reported? Did you test the older version with add-ons disabled? If not, could you do it now and confirm that ctrl+c, ctrl+v, etc. are reported?
2. Word 2019 raises UIA notifications for these shortcuts/actions and Word 2016 doesn't. @josephsl would you confirm or infirm this point?

Note that even if this issue affects Word shortcuts, it is very unlikely related to the new capability of reporting indents.
